src/chessground/render.ts

```diff
@@ -264,12 +264,35 @@ function addSquare(squares: Map<Key, string>, key: Key, klass: string) {
 
 function computeSquareClasses(d: State): Map<Key, string> {
```

Comment:
It is quite long now. What about splitting into smaller functions

```

Reply:
Let's split it in the future, maybe after adding more indications